content/en/platform/corda/1.5/cenm/_index.md

```diff
@@ -61,64 +61,64 @@ For a quick start guide on deploying Corda Enterprise Network Manager services a
 * [Certificate Revocation List]({{< relref "../../../../../en/platform/corda/1.5/cenm/certificate-revocation.md" >}})
 
 ## CENM releases
+[](../../1.5/cenm/release-notes.md)
```

Comment:
Looks like it entered a double line for the same file (can delete the first)

Reply:
If this is the only comment @paulmoloneyr3 
Please have someone else from the team review too and let's do the change and merge it instead of waiting for @colin-hillis to return from PTO

Reply:
Hi @yarivus , just fixed that and pushed.